Sources/Brave/Frontend/Browser/New Tab Page/Sections/BraveNewsSectionProvider.swift

```diff
@@ -70,6 +72,8 @@ class BraveNewsSectionProvider: NSObject, NTPObservableSectionProvider {
     collectionView.register(FeedCardCell<BraveNewsEmptyFeedView>.self)
     collectionView.register(FeedCardCell<HeadlineCardView>.self)
     collectionView.register(FeedCardCell<SmallHeadlinePairCardView>.self)
+    collectionView.register(FeedCardCell<SmallRateCardView>.self)
```

Comment:
This card doesn't need to be registered since its only shown inside of `SmallHeadlineRatePairCardView`